client/src/Movies/SavedList.js

```diff
@@ -10,9 +11,19 @@ export default class SavedList extends Component {
       <div className="saved-list">
         <h3>Saved Movies:</h3>
         {this.props.list.map(movie => (
+          <NavLink to={`/movies/${movie.id}`} key={movie.id}>
           <span className="saved-movie">{movie.title}</span>
+          </NavLink>
         ))}
-        <div className="home-button">Home</div>
+
+        <Link to={"/App"}>
+          <div className="home-button">Home</div>
+        </Link>
+
+        <Link to={"/"}>
+          <div className="home-button">Movie List</div>
+        </Link>
+
```

Comment:
So I'm not sure what the stretch was, but looking at this, I'm guessing that you Link and NavLink usage is reversed. The only difference between the two is that NavLink allows you to add styling to the link if it is selected. This means that you could have each link look kind of like a tab, and then if you're on `"/"`, the Movie List button will be highlighted, and if you're on `"/App"`, the Home button will be highlighted.

Looking at your code, I'm thinking you never quite finished this. Do you want to touch base and talk about the implementation at all? Let me know. Overall, great work! 👍